content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Agent Configuration
+weight: 45
+spelling: cSpell:ignore distro mkdir uninstrumented virtualenv
+---
+
+Automatic instrumentation with Python uses a Python agent binary that can be attached to any Python application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks.
+
+## Setup
+
+Run the following commands to install the appropriate packages.
+
+```console
+pip install opentelemetry-distro \
+	opentelemetry-exporter-otlp
+
+opentelemetry-bootstrap -a install
+```
+
+The `opentelemetry-distro` package installs the API, SDK, and the `opentelemetry-bootstrap` and `opentelemetry-instrument` tools.
+
+The `opentelemetry-bootstrap -a install` command reads through the list of packages installed in your active `site-packages` folder, and installs the corresponding auto-instrumentation libraries for these packages, if applicable. For example, if you already installed the `flask` package, running `opentelemetry-bootstrap -a install` will install `opentelemetry-instrumentation-flask` for you.
+
+> **NOTE:** If you leave out `-a install`, the command will simply list out the recommended auto-instrumentation packages to be installed.
+More information can be found [here](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation#opentelemetry-bootstrap).
+
+## Configuring the agent
```

Comment:
Additionally, you could move the configuration part from automatic/_index.md into this chapter (to have it only one place) and add a link to _index.md so that users find this page.